docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -12,29 +12,51 @@ Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
 we recommend using ADLS Gen2. Learn more from `the official documentation
 <https://docs.microsoft.com/en-us/azure/data-lake-store/data-lake-store-overview>`_.
 
-Hive connector configuration
-----------------------------
+Hive connector configuration for Azure Storage credentials
+----------------------------------------------------------
+
+All of the configuration properties for the Azure storage driver are stored in
+the Hadoop ``core-site.xml`` configuration file. To configure Trino to use the
+Azure Storage credentials, set the following configuration properties in the
+catalog properties file. It is best to use this type of configuration if the
+primary storage account is linked to the cluster.
+
+When there are secondary storage accounts involved, we recommend configuring
+Trino using a ``core-site.xml`` containing the appropriate credentials for each
+account.
 
-All configuration for the Azure storage driver is stored in the Hadoop
-``core-site.xml`` configuration file. The path to the file needs to be
-configured in the catalog properties file:
+The path to the file must be configured in the catalog properties file:
 
 .. code-block:: text
 
     hive.config.resources=<path_to_hadoop_core-site.xml>
 
-Configuration for Azure Storage credentials
--------------------------------------------
+Example
```

Comment:
@findinpath and @mfzarko how's this? I created an example section with your suggestion. 

@mosabua from a structure standpoint, do you agree with the example subsection style?

Reply:
LGTM